app/tests/api/api_v1/test_test_run_executions.py

```diff
@@ -796,6 +796,25 @@ async def test_test_run_execution_start(async_client: AsyncClient, db: Session)
     assert content["id"] == test_run_execution.id
 
 
+@pytest.mark.asyncio
+async def test_test_run_execution_start_no_pics(
+    async_client: AsyncClient, db: Session
+) -> None:
+    test_run_execution = create_test_run_execution_with_some_test_cases(db=db, pics={})
+
+    # First attempt to start test run
+    response = await async_client.post(
+        f"{settings.API_V1_STR}/test_run_executions/{test_run_execution.id}/start",
+    )
+
+    # Assert 200 OK and that test run data is returned
```

Comment:
I believe that this comment is incorrect, because the expected status code is not 200 OK and no test run data should be returned